### ISSUE TITLE
ESP32 use?

### ISSUE THREAD

**Issue by andburch** [pushdata-io/Arduino_ESP8266_SSL#4]
This probably isn't the place to ask this, but is there a way to adapt this to work with the ESP32?

**Comment by ragnarlonn** [comment]
@andburch Yes, that shouldn't be so difficult. I'd suspect you can just take Pushdata_ESP8266_SSL.h and rename it to Pushdata_ESP32_SSL.h, then change line 22-24 from:

```cpp
#ifdef ESP8266
#include "ESP8266WiFi.h"
#include "ESP8266WiFiMulti.h"
```

...to instead include the corresponding ESP32 include files, when "ESP32" is #defined. Looking at https://github.com/espressif/arduino-esp32/blob/master/libraries/WiFi/src/WiFiMulti.h it seems it is very similar to the ESP8266 version (the comments actually wrongly state that the file *is* ESP8266WifiMulti.h)

Then you might have to remove line 158 with the call to `system_update_cpu_freq()` as I'm not sure that call is available on ESP32.

If it sounds tricky I can take a look at it later but I can't say when I'll have the time and I currently don't have en ESP32 to test it on, I think, so I'll just be able to make sure it compiles.

**Comment by andburch** [comment]
Actually, the POST commands are so darn easy, I managed (so far) to do what I want without this library. I tried modifying it as you said, but I think there are too many smaller references that have to change, etc.
Regardless, I figured out a solution for me!
